### PR TITLE
PB-636: Adding viewport documentation

### PR DESCRIPTION
**Pull request by bdenham**
## Description

This PR provides documentation on How viewports work in preparation for the next topic, How to add viewports.

Main illustration:

image

### CODE REVIEW

File: docs/viewports/how-viewports-work.md [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+</var>
+```
+
+Configuration data descriptions:
+
+- `Magento_PageBuilder` - Defines the module to which viewport data applies.
+- `breakpoints` - Defines `viewports` object for the module.
+- `desktop` and `mobile` - Defines viewport objects named `desktop` and `mobile`.
+- `label` - (string) Defines title of tooltip.
+- `stage` - (bool) ????
```
Comment (omiroshnichenko) [inline docs/viewports/how-viewports-work.md:203]: Display on stage or npt

File: docs/viewports/how-viewports-work.md [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+- `default` - (bool) Determines if the viewport is selected when the stage is loaded.
+- `class` - (string) CSS class to apply to the viewport switcher button (not the stage).
+- `icon` - (string) - URL to the button image (SVG) location.
+- `media` - (string) ????
```
Comment (omiroshnichenko) [inline docs/viewports/how-viewports-work.md:207]: media query rule that used in the admin area

File: docs/viewports/how-viewports-work.md (hunk 1 above)
Comment (bdenham) [reply docs/viewports/how-viewports-work.md:203]: That's what I thought, but I couldn't get it to work recently. Something just screwed up in my example probably. thanks!

File: docs/viewports/how-viewports-work.md (hunk 2 above)
Comment (bdenham) [reply docs/viewports/how-viewports-work.md:207]: Also what I thought, but couldn't see differences when I changed it in my example. Again, probably something happened in my example.

**Pull request closed without merging**